edk2basetools/VfrCompiler/IfrCommon.py

```diff
@@ -0,0 +1,100 @@
+## @file
+# This file is used to define common funtions.
+#
+# Copyright (c) 2022-, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+import Common.EdkLogger as EdkLogger
+from VfrCompiler.IfrCtypes import EFI_GUID
+from Common.BuildToolError import PARAMETER_INVALID
+
+# Enumeration of EFI_STATUS.
```

Comment:
Many tools are going to need this.  I would suggest figuring out a common way to do this.  See https://github.com/tianocore/edk2-pytool-library/blob/master/edk2toollib/uefi/status_codes.py as an example.